### PR TITLE
Set line-height to inherit

### LINKED ISSUE #536: Change line-height from normal to inherit
**styleguide.config.js**
```js
const path = require('path');

module.exports = {
  title: 'BSA React Web Components',
  sections: [
    {
      name: 'Introduction',
      content: 'docs/Introduction.md',
    },
    {
      name: 'Reboot',
      content: 'docs/Reboot.md',
    },
    {
      name: 'Components',
      components: 'src/**/[A-Z]*.js',
    },
  ],
  require: [
    path.join(__dirname, 'docs/assets/styles/index.less'),
  ],
  assetsDir: 'docs/assets',
  webpackConfig: {
    module: {
      loaders: [
        {
          test: /\.js$/,
          exclude: /node_modules/,
          loader: 'babel-loader',
          query: {
            presets: ['es2015', 'stage-3', 'react'],
          },
        },
        {
          test: /\.css$/,
          loaders: ['style-loader', 'css-loader'],
        },
        {
          test: /\.less$/,
          use: [
            'style-loader',
            'css-loader',
            {
              loader: 'less-loader',
              options: {
                sourceMap: true,
              },
            },
          ],
        },
      ],
    },
  },
};
```

webpack version: ^3.0.0
OS, browser version: OS X, Chrome 58.0.3029.110 (64-bit)

Hi! I like react-styleguidist a lot.

I'm trying to set up a page where I showcase how headers, lists and other elements look by importing the library (and its stylesheet).

Currently it looks like this 

screen shot 2017-07-07 at 3 40 44 pm

If I change [this div](https://github.com/styleguidist/react-styleguidist/blob/a010018f6672af81332018399fc0e595e81d4b24/src/rsg-components/Playground/PlaygroundRenderer.js#L26) `line-height` from `normal` to `inherit` it looks how I would expect it to.

screen shot 2017-07-07 at 3 43 34 pm

Could we change it so the default style for line-height is inherit?

### PR DESCRIPTION
**Pull request by SaraVieira**
Closes #536

### CODE REVIEW

**Comment by SaraVieira** [comment]
@sapegin Changed ! Thanks , it makes more sense there :)

**Comment by sapegin** [comment]
I hope it won’t break the UI ;-)

File: src/styles/nonInheritedProps.js [hunk 1]
```diff
@@ -57,5 +57,5 @@ export default {
 	'unicode-bidi': 'normal',
 	'vertical-align': 'baseline',
 	'width': 'auto',
-	'z-index': 'auto',
+	'z-index': 'auto'
```
Comment (sapegin) [inline src/styles/nonInheritedProps.js]: This change is weird ;-)

**Comment by SaraVieira** [comment]
From what I checked it didn't seen to break :)

A good idea maybe would be to put something like netifly on PR's always mounting one of the instances so we can always be sure :)

cc @sapegin

**Comment by sapegin** [comment]
@SaraVieira Could you take a look at my last comment? Is it Prettier or some accidental change?

**Comment by sapegin** [comment]
> A good idea maybe would be to put something like netifly on PR's always mounting one of the instances so we can always be sure :)

Yeah, we use Netlify for the site anyway ;-)

File: src/styles/nonInheritedProps.js (hunk 1 above)
Comment (SaraVieira) [inline src/styles/nonInheritedProps.js]: I thought I answered this 😢 

I think this was prettier should I remove it ?

**Comment by SaraVieira** [comment]
@sapegin Can I do this ? Never used any GH integration so it would be a cool experience :)

**Comment by sapegin** [comment]
@SaraVieira You can at least think how to do it, I’m not really sure myself ;-) Could you open a new issue?

**Comment by SaraVieira** [comment]
Sure! As soon as I get home :)

File: src/styles/nonInheritedProps.js (hunk 1 above)
Comment (sapegin) [inline src/styles/nonInheritedProps.js]: I was wandering why Prettier didn’t put it back ;-) Better to revert it ;-)

**Comment by SaraVieira** [comment]
@sapegin All changes made and creating the issue now

File: src/styles/setupjss.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ jss.setup({
 
 				// Allow inheritance because it may be set on body and should be available for user components
 				fontFamily: 'inherit',
+				'line-height': 'inherit',
```
Comment (sapegin) [inline src/styles/setupjss.js]: Sorry, just noticed: it should be `lineHeight` of course.

Comment (SaraVieira) [inline src/styles/setupjss.js]: Also fixed 👍

**Pull request merged**